src/components/Modal/Modal.tsx

```diff
@@ -36,21 +34,17 @@ const Content = styled.div`
     overflow: auto;
     padding: 1em 1.5em 1.5em;
     width: 100%;
+    color: ${(props) => props.theme.main.foreground[1]};
+    background-color: ${(props) => props.theme.main.background[1]};
 `;
 
 const CloseButton = styled.button.attrs({
```

Comment:
Could we also get a cursor: pointer on the close button? Whoever did the modal forgot to add it.